dashboard/signals_test.py

```diff
@@ -0,0 +1,36 @@
+"""
+Tests for signals
+"""
+
+from django.contrib.auth.models import User
+
+from dashboard.models import ProgramEnrollment, CachedEnrollment
+from courses.models import Program, Course, CourseRun
+from search.base import ESTestCase
+
+
+# pylint: disable=no-self-use
+class SignalProgramEnrollmentTest(ESTestCase):
+    """
+    Test class for signals that creates a profile whenever a user is created
+    """
+    def test_program_enrollment_creation(self):
+        """
+        Tests that if a user is created, the profile for such user is created
+        """
+        # there are no users
+        assert len(ProgramEnrollment.objects.all()) == 0
+        assert len(CachedEnrollment.objects.all()) == 0
+
+        # the creation of a user triggers the creation of a profile
+        user = User.objects.create(username='test', password='test')
+        program = Program.objects.create(title='New Program', live=True)
+        course = Course.objects.create(program=program, title="Test Course")
+        course_run = CourseRun.objects.create(course=course, title="Test Run")
+        enrollment = CachedEnrollment.objects.create(user=user, course_run=course_run, data=None)
+        assert len(ProgramEnrollment.objects.all()) == 0
+        enrollment.data = {}
+        enrollment.save()
```

Comment:
does this work? line 11 in `dashboard/signals.py` (`if instance.data:`) makes it look like this shouldn't work

```python
# Doesn't print anything
if {}:
     print('hi')

# Prints 'hi'
if {'a':1}:
     print('hi')
```